### PR TITLE
add None as a possible value for DebugInfoFlags

### PR DESCRIPTION
**Pull request by bashbaug**
It's possible that no debug info flags are needed, in which case the debug info flags will have value `0`.  This case should be disassembled into `None` and then `None` should be assembled back into `0`.  Without this change many SPIR-V modules with debug information cannot be properly disassembled and assembled.

I think this may require a minor spec update as well?  I'll work on that next.

CC: @AlexeySotkin

### CODE REVIEW

**Comment by bashbaug** [comment]
Hmm, I may need to regenerate the headers for these extended instruction sets also?  I'll do this shortly.

**Review by AlexeySotkin — APPROVED** [review]
LGTM, thanks!

**Comment by bashbaug** [comment]
Ping @dneto0 @johnkslang @raunraun - could one of you please review and merge this PR assuming everything looks good?  It's holding up some CTS tests.  Thanks!

**Comment by iamen12345** [comment]
> It's possible that no debug info flags are needed, in which case the debug info flags will have value `0`. This case should be disassembled into `None` and then `None` should be assembled back into `0`. Without this change many SPIR-V modules with debug information cannot be properly disassembled and assembled.
> 
> I think this may require a minor spec update as well? I'll work on that next.
> 
> CC: @AlexeySotkin

hzjxnxjx

**Pull request merged**
